src/main/resources/permissions/system-user-permissions.csv

```diff
@@ -67,3 +67,5 @@ inventory-storage.alternative-title-types.item.delete
 circulation-storage.cancellation-reasons.item.post
 circulation-storage.cancellation-reasons.item.put
 circulation-storage.cancellation-reasons.item.delete
+user-settings.custom-fields.item.post
+user-settings.custom-fields.collection.get
```

Comment:
Karate test should be updated also
we check number of system users' permisdions in two places: L123 and L233.
Example: https://github.com/folio-org/folio-integration-tests/pull/1042/files